### ISSUE TITLE
It should take into account configurations from "cypress.json"

### ISSUE THREAD

**Issue by 6220119** [LironEr/cypress-mochawesome-reporter#13]
Great works 👍 
I would like to ask for a feature:

**Is your feature request related to a problem? Please describe.**
In our project, we are using the following config from `cypress.json` file:
- `screenshotsFolder`
- `videosFolder`

This plugin should also take these configs into account together with cypress default and CLI arguments.

**Describe the solution you'd like**
- The reporter should also read from `cypress.json` file
- Additionally, it should also check the `reporterOptions` (for `mochawesome`) from `cypress.json`

**Describe alternatives you've considered**
- Use CLI arguments to override

**Additional context**
- None

**Comment by LironEr** [comment]
The reporter already knows how to deal with different screenshots folder.

So for example if you set up your `cypress.json` like:
```
{
  "reporter": "cypress-mochawesome-reporter",
  "screenshotsFolder": "images"
}
```

You will need to send `--screenshotsDir` flag to `generate-mochawesome-report` CLI

```
generate-mochawesome-report --screenshotsDir images
```

[Full example](https://github.com/LironEr/cypress-mochawesome-reporter/tree/master/examples/screenshots-folder) 

Hope that answers your issue, if not please replay 😊

**Comment by mortik** [comment]
What @6220119 probably meant was it would be cool if i don't need to run ```generate-mochawesome-report --screenshotDir images``` with the --screenshotDir argument but instead reuse the settings from cypress.json

**Comment by LironEr** [comment]
Yeah, you probably right
Will have something ready in a day or two 👍

**Comment by LironEr** [comment]
Released in [v1.3.0](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v1.3.0)

Thanks @6220119 @mortik

**Comment by mortik** [comment]
Awesome thank you :)
